src/components/change-wallet/AddressRow.js

```diff
@@ -29,7 +29,6 @@ const NOOP = () => undefined;
 
 const sx = StyleSheet.create({
   accountLabel: {
-    fontFamily: fonts.family.SFProRounded,
```

Comment:
If wallet doesn't have a name, it is shown with wrong font (default Roboto) on Android. This seems to fix the problem.